staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-mobile.css

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
HINT main-mobile.css is a bit redundant name. let's keep it simple, just mobile.css is enough. rename it.